modular_skyrat/modules/opposing_force/code/equipment/equip_parent.dm

```diff
@@ -1,3 +1,22 @@
+/// Global list of equipment parents, used for unit testing. PLEASE PLEASE PLEASE UPDATE THIS WITH NEW CATEGORIES AS THEY GET ADDED
```

Comment:
I do not like having hard coded lists like this. People shouldn't have to add it he4e too tbh.

Reply:
This list was the product of me trying to find a not-expensive method of getting only the direct subtypes of /datum/opposing_force_equipment, will give it another shot later

Reply:
You only need to generate the list once during server initialization (probably during SSopposing_force startup). `subtypesof()` isn't _that_ expensive.

Reply:
Found a better way to do this!